pyrokinetics/pyroscan.py

```diff
@@ -129,6 +153,10 @@ def write(self, file_name=None, base_directory=None, template_file=None):
                 for run_dir in self.pyro_dict.keys()
             ]
 
+        # Dump json file with pyroscan data
+        json_file = os.path.join(self.base_directory, "pyroscan.json")
+        json.dump(self.pyroscan_json, open(json_file, "w+"))
```

Comment:
This won't `close` the file which might result in the contents not getting written to disk. It's better to always use `with open` to ensure the file gets closed

```suggestion
        with open(json_file, "w+") as f:
            json.dump(self.pyroscan_json, f)
```

Reply:
Also for the loading of the json?

Reply:
Yep! I didn't spot that at first